src/commands/createNewProject/ProjectCreateStep/DotnetProjectCreateStep.ts

```diff
@@ -41,7 +41,12 @@ export class DotnetProjectCreateStep extends ProjectCreateStepBase {
         }
         const functionsVersion: string = 'v' + majorVersion;
```

Comment:
cc @liliankasem 

For Alex's context, this is a new change by Lilian to the .NET isolated templates. PR here: https://github.com/Azure/azure-functions-templates/pull/1240. Basically, in the past the isolated worker only really supported one .NET version at a time (or at least only one we cared about for the sake of templates). Now, they support net48, net6.0, and net7.0 all at once which is why a new parameter was introduced. If the arg is not passed, it will currently default to net6.0.

Also - it's important to note that the version of the `dotnet` cli being used to create the project does not necessarily match the version of the project. I can have all three versions installed on my machine but I should still be able to pick which one I use for my project.